composer.json

```diff
@@ -17,17 +17,19 @@
         }
     ],
     "require": {
-        "php": ">=7.1"
+        "php": ">=7.1",
+        "symfony/event-dispatcher": "~2.5"
```

Comment:
I know it's not purpose of this PR but I need to remind that since this is now hard requirement, this constraint needs to be opened up more now. It doesn't allow symfony/event-dispatcher >= 2